src/wallet3/wallet.cpp

```diff
@@ -71,6 +71,7 @@ namespace wallet
   void
   Wallet::init()
   {
+    keys->expand_subaddresses({50, 200});
```

Comment:
This needs to be configurable; some wallets might need a larger lookahead, but also hardware-wallet backed wallets use a much smaller lookahead (because more subaddresses has a noticeable performance drag when using the hardware wallet).